plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/recipes/UpgradeJenkinsTestHarnessVersion.java

```diff
@@ -0,0 +1,66 @@
+package io.jenkins.tools.pluginmodernizer.core.recipes;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Option;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.maven.ChangePropertyValue;
+import org.openrewrite.maven.MavenIsoVisitor;
+import org.openrewrite.maven.RemoveRedundantDependencyVersions;
+import org.openrewrite.xml.tree.Xml;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class UpgradeJenkinsTestHarnessVersion extends Recipe {
+    /**
+     * Logger
+     */
+    private static final Logger LOG = LoggerFactory.getLogger(UpgradeJenkinsTestHarnessVersion.class);
+
+    /**
+     * The latest compatible version.
+     */
+    @Option(displayName = "Version", description = "The version.", example = "2.70")
+    String latestCompatibleVersion;
```

Comment:
If we have this information on the JDK can we get rid of this option?

If we use somethink like

```java
PluginMetadata pomMetadata = new PomResolutionVisitor().reduce(tree, new PluginMetadata());

// Can be an utils on JDK? That take a Jenkins version and return directly the latest test-harness compatible?
String latestTestHarnessVersion = JDK.min(new HashSet<>(JDK.get(pomMetadata.getJenkinsVersion())));
```